src/unstable-temp/DownloadImageModal/ImageForm.tsx

```diff
@@ -238,6 +255,46 @@ export const ImageForm = ({
 		[actions, downloadOptions],
 	);
 
+	const flashWithEtcher = (
+		_event: React.MouseEvent,
+		model: DownloadOptions,
+	) => {
+		const modelCopy = { ...model };
+		// TODO check if possible to edit Etcher to avoid a double encode on the version.
+		if (modelCopy.version) {
+			modelCopy.version = encodeURIComponent(modelCopy.version);
+		}
+		if (modelCopy.network === 'ethernet') {
+			modelCopy.wifiSsid = undefined;
+			modelCopy.wifiKey = undefined;
+		}
+		const queryParams = Object.entries(modelCopy)
+			.map(([key, value]) => (!!value ? `${key}=${value}` : null))
+			.filter((param) => !!param)
+			.join('&');
+		const imageUrl = `${downloadUrl}?${queryParams}`;
+		const axiosConfig = {
+			method: 'POST',
+			url: imageUrl,
+			...(authToken && {
+				headers: {
+					Authorization: `Bearer ${authToken}`,
+				},
+			}),
+			data: pickBy(modelCopy, identity),
```

Comment:
NIT: How about avoiding the import?
```suggestion
			data: pickBy(modelCopy, (value) => !!value),
```